workloads/kube-burner/workloads/cluster-density/cluster-density.yml

```diff
@@ -0,0 +1,100 @@
+---
+global:
+  writeToFile: false
+  indexerConfig:
+    enabled: {{.INDEXING}}
+    esServers: ["{{.ES_SERVER}}"]
+    insecureSkipVerify: true
+    defaultIndex: {{.ES_INDEX}}
+    type: elastic
+  measurements:
+    - name: podLatency
+      esIndex: {{.ES_INDEX}}
+jobs:
+  - name: cluster-density
+    jobIterations: {{.JOB_ITERATIONS}}
```

Comment:
Should this be {{.JOB_ITERATIONS}} or {{.TEST_JOB_ITERATIONS}}. Not 100% sure where it gets invoked?

Reply:
JOB_ITERATIONS is passed as env var to the kube-burner job in the file workloads/kube-burner/kube-burner-crd.yaml, Kube-burner now reads that env var.